### PR TITLE
Added support for the align option from image directive

### PR DESCRIPTION
**Pull request by nicolasmartin3d**

### CODE REVIEW

File: sphinxcontrib/images.py [hunk 1]
```diff
@@ -80,6 +80,15 @@ class ImageDirective(Directive):
     It's backward compatibile and it's adding more cool stuff.
     '''
 
+    align_h_values = ('left', 'center', 'right')
+    align_values = align_h_values
```
Comment (spinus) [inline sphinxcontrib/images.py]: Do we need two variables here? I see that align_h_values is used only once, in this line.

File: sphinxcontrib_images_lightbox2/lightbox2.py [hunk 2]
```diff
@@ -35,10 +35,12 @@ def visit_image_node_html(self, writer, node):
                 ))
         writer.body.append(
             '''<img src="{src}"
+                    class="{cls}"
                     width="{width}"
                     height="{height}"
                     alt="{alt}"/>
                     '''.format(src=node['uri'],
+                               cls='align-%s' % node['align'],
```
Comment (spinus) [inline sphinxcontrib_images_lightbox2/lightbox2.py]: As I'm not sure I ask, do you know sphinx has classes align-* or we have to add css to support this?

File: sphinxcontrib/images.py (hunk 1 above)
Comment (nicolasmartin3d) [inline sphinxcontrib/images.py]: No, not really ! I am fixing that.

File: sphinxcontrib_images_lightbox2/lightbox2.py (hunk 2 above)
Comment (nicolasmartin3d) [inline sphinxcontrib_images_lightbox2/lightbox2.py]: Yes sphinx already support those, this is how an  (with the Image directive) is aligned.

**Pull request merged**